Beeper/BarcelonaMautrixIPC/Runtime/BLStandardIO.swift

```diff
@@ -65,6 +65,16 @@ public extension FileHandle {
     }
 }
 
+extension FileHandle: MautrixIPCInputChannel {
+    public func listen(_ cb: @escaping (Data) -> ()) {
+        self.handleDataAsynchronously(cb)
+    }
+}
+
+extension FileHandle: MautrixIPCOutputChannel {
+    
```

Comment:
There should be something else here - there's a method that `FileHandle` has to implement to conform to this protocol, so if you implemented it yourself, could you move it to here? Or maybe add a comment explaining if it's already implemented by `FileHandle` or something like that?